src/main/warpscript/io.warp10/warp10-ext-barcode/BARCODE.mc2

```diff
@@ -54,11 +54,11 @@ BARCODE
     '>
   ]
   'conf' [ 'warpscript.extensions' ]
-  }
+  } 'info' STORE
 <%
   !$info INFO
-  SAVE 'context' STORE
-    <% BARCODE %>
+  SAVE 'context' 
```

Comment:
either remove 'context' or leave the STORE

Reply:
big fingers ....